pkgs/applications/science/medicine/dcmtk/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, zlib, libtiff, libxml2, openssl, libiconv, libpng, cmake }:
+
+with stdenv.lib;
+stdenv.mkDerivation {
+  pname = "DCMTK";
+  version = "3.6.4";
+  src = fetchFromGitHub {
+      owner = "DCMTK";
+      repo = "dcmtk";
+      rev = "1967b13134308f311e6a827e616958c6a4da5bc9";
+      sha256 = "0fbx35zax8n4gayaac5bankqwzg2y2adggykbbf8lpd773jfxsp6";
+  };
+
+  buildInputs = [ cmake libpng zlib libtiff libxml2 openssl libiconv ];
+
+  meta = {
+    description = "DICOM Toolkit";
+    longDescription = ''
+      DCMTK is a collection of libraries and applications implementing large parts of the DICOM standard.
+      It includes software for examining, constructing and converting DICOM image files, handling offline media, sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.
```

this line is too long
```suggestion
      It includes software for examining, constructing and converting DICOM image files, handling offline media, 
      sending and receiving images over a network connection, as well as demonstrative image storage and worklist servers.
```